insights/parsers/df.py

```diff
@@ -289,5 +313,24 @@ class DiskFree_AL(DiskFree):
                 Available      available
                 Use%           capacity
                 Mounted on     mounted_on
+        block_size (str): The unit of display values, '1K' by default.
```

Comment:
@xiangce 
As sosreport is using `df_al`, we can check `block_size` in rules to workaround this.       
For this point, do we consider to provide an `int` value of `block_size` at same time?  So we do the B/K/M/G converting here instead in each rule that use this parser.   
Like: 
```suggestion
        block_size_raw (str): The unit of display values, '1K' by default.
        block_size (int): The calculatable value of block_size_raw in byte, 1024 for '1K', 1048576 for '1M'.
```

Reply:
@JoySnow - Updates applied, please have a look.